release-notes/schema/dotnet-required-packages.json

```diff
@@ -0,0 +1,170 @@
+{
+  "description": "The set of packages required by a given .NET version for a set of distros.",
+  "type": [
+    "object",
+    "null"
+  ],
+  "properties": {
+    "ChannelVersion": {
+      "type": "string"
+    },
+    "EolDate": {
+      "type": "string",
+      "format": "date"
+    },
+    "Packages": {
+      "type": "array",
+      "items": {
+        "description": "A logical package that will exist in various distros with different package names. Includes the scenarios for which the package is required.",
```

Comment:
What is meant by a 'logical package'?

Reply:
I was also thinking of "nominal package". Open to better terms.

I was trying to do few things:

- Provide additional information on each package in one place, per DRY rules.
- Provide a canonical name for each package since most package names differ by distro and/or distro version.
- Make it possible to write a single script that works across distros w/o having any knowledge of the particular distro and their package names.

Reply:
I like nominal package better! I think saying 'The canonical name for the package that will exist...' works great, but unfortunately there are 2 relevant interpretations of Canonical in this instance. Perhaps 'statutory package name'? 😄

Reply:
Yup. The double meaning of "canonical" definitely came to mind.